src/rust/ide/view/graph-editor/src/component/visualization/container/visualization_chooser.rs

```diff
@@ -0,0 +1,309 @@
+//! Definition of the icon + menu displayed to choose the visualisation for a
+//! node.
+
+use crate::prelude::*;
+
+use crate::component::visualization;
+
+use enso_frp as frp;
+use enso_frp;
+use ensogl::application::Application;
+use ensogl::data::color;
+use ensogl::display::shape::*;
+use ensogl::display::traits::*;
+use ensogl::display;
+use ensogl::gui::component;
+use ensogl_gui_list_view as list_view;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const HOVER_COLOR    : color::Rgba = color::Rgba::new(1.0,0.0,0.0,0.000_001);
```

Comment:
Duplicate, also please use theme manager for that, and show how it looks in light mode.